medusa/server/web/home/handler.py

```diff
@@ -1293,8 +1293,8 @@ def check_show_for_language(show_obj, language):
         indexer_api_params = indexerApi(show_obj.indexer).api_params.copy()
         indexer_api_params['language'] = language
         indexer_api_params['episodes'] = False
-        t = indexerApi(show_obj.indexer).indexer(**indexer_api_params)
         try:
+            t = indexerApi(show_obj.indexer).indexer(**indexer_api_params)
```

Comment:
Personally this  is the best solution as I have it now. As the creating the indexer_api can raise IndexerUnavailable. But chose to use IndexerException, as that's a somewhat broader catch. I don't want to give tvdb the room to prevent me from editing a show.

Also the IndexerException is around the check_show_for_language method, because the creation of the api aws well as the api call itself could in theory raise an exception other then IndexerShowNotFoundInLanguage.